vector/src/main/java/im/vector/riotredesign/features/home/room/list/UnreadCounterBadgeView.kt

```diff
@@ -29,24 +29,24 @@ class UnreadCounterBadgeView : AppCompatTextView {
 
     constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr)
 
-    fun render(count: Int, highlighted: Boolean) {
-        if (count == 0) {
+    fun render(state: State) {
+        if (state.count == 0) {
             visibility = View.INVISIBLE
         } else {
             visibility = View.VISIBLE
-            val bgRes = if (highlighted) {
+            val bgRes = if (state.highlighted) {
```

Comment:
Could be a rounded rectangle instead of oval

Reply:
done